src/Microsoft.AspNetCore.Server.Testing/Deployers/IISExpressDeployer.cs

```diff
@@ -34,21 +34,23 @@ public override DeploymentResult Deploy()
                 DotnetPublish();
             }
 
+            var webRootLocation = DeploymentParameters.PublishApplicationBeforeDeployment ? DeploymentParameters.PublishedApplicationRootPath : DeploymentParameters.ApplicationPath;
```

Comment:
- I glanced through the tests and do not see this property being used
- From the documentation here https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Server.Testing/Common/DeploymentResult.cs#L18-L21, it clearly doesn't indicate about `wwwroot` but i agree `AppRoot` is a better name